src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3469,7 +3470,7 @@
             GeneratedJavaTokenTypes.BLOCK_COMMENT_BEGIN;
 
     /**
-     * End of block comment: '* /'.
+     * End of block comment: '&#42;/'.
```

Comment:
Why can't this remain as `*`?

Reply:
because this token is '*/' not '* /'.

Reply:
I see now, thanks.
This can be ignored.